backend/models/requiredDetails.js

```diff
@@ -0,0 +1,31 @@
+const Sequelize = require('sequelize');
+const db = require('../config/database');
+
+//User model associated with Database User table
+const RequiredDetails = db.define(
+  'RequiredDetails',
+  {
+    Id: {
+      primaryKey: true,
+      autoIncrement: true,
+      type: Sequelize.INTEGER,
+    },
+    WeightRequired: {
+      type: Sequelize.BOOLEAN,
+    },
+    TemperatureRequired: {
+      type: Sequelize.BOOLEAN,
+    },
+    SymptomsRequired: {
+      type: Sequelize.BOOLEAN,
+    },
+    Patient_PatientId: {
```

Comment:
Should we change this to PatientId instead of Patient_PatientId ?

Reply:
As discussed, it will be left as it is for now